src/frontend/components/ConverseInitializer/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Spinner } from 'grommet';
+import React, {
+  Fragment,
+  PropsWithChildren,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import { Video } from 'types/tracks';
+import { converseMounter } from 'utils/conversejs/converse';
+import { converse } from 'utils/window';
+
+interface ConverseMonterProps {
+  video: Video;
+}
+
+export const ConverseInitializer = ({
+  children,
+  video,
+}: PropsWithChildren<ConverseMonterProps>) => {
+  const initConverse = useMemo(() => {
+    return converseMounter();
+  }, [converseMounter]);
+  const [isConverseLoaded, setIsConverseLoaded] = useState(
+    converse !== undefined,
+  );
+
+  useEffect(() => {
+    const eventHandler = () => {
+      setIsConverseLoaded(true);
+    };
+    if (!isConverseLoaded) {
+      window.addEventListener('converse-loaded', eventHandler);
+    }
+    return () => {
+      window.removeEventListener('converse-loaded', eventHandler);
+    };
+  }, [isConverseLoaded]);
+
+  useEffect(() => {
+    if (isConverseLoaded && video.xmpp) {
```

Comment:
In the `Chat` component, the initialization was based on the `live_state`, not on the `video.xmpp` property. Wy did you change this ?

Reply:
I did it firstly to remove the assertion from previous implementation and I thought we talked about that and XMPP fields should be available only when live_state is not idle

Reply:
Indeed and in fact this old check was a legacy from when we allowed multiple values in `live_state`. Check `xmpp` is better :+1:

Reply:
I apologize but my assertion is false. The `video.xmpp` property is set when `video.live_state` is not null. You have to keep the old check with `video.live_state !== idle`